cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -186,12 +191,19 @@ public void setVoltageAngleNodeBreaker() {
                 return;
             }
             LOG.warn(
-                    "Can't find a calculated Bus to set Voltage, Angle, but found a configured Bus {}. Connectivity node {}",
-                    bus, id);
+                "Can't find a calculated Bus to set Voltage, Angle, but found a configured Bus {}. Connectivity node {}",
```

Comment:
Not related to this PR, but CalculatedBus and ConfiguredBus are implementation specific and should not appear in this message

Reply:
What error message do you suggest?

Reply:
Somethink like that:
> Can't find a bus from the Bus View to set Voltage and Angle, we use the bus {} from the Bus/Breaker view. Connectivity node {}